package.json

```diff
@@ -69,13 +70,14 @@
         "eleventy-plugin-nesting-toc": "^1.3.0",
         "eleventy-plugin-page-assets": "^0.3.0",
         "eleventy-plugin-reading-time": "^0.0.1",
-        "eslint": "^8.19.0",
+        "eslint": "^8.56.0",
         "eslint-config-eslint": "^7.0.0",
```

Comment:
Let's update it by the way

Reply:
Updated, thanks for the catch! Somehow I missed this.